app/src/main/res/layout/fragment_password.xml

```diff
@@ -2,32 +2,38 @@
 <android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    xmlns:app="http://schemas.android.com/apk/res-auto">
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
 
     <android.support.design.widget.TextInputLayout
         android:id="@+id/layout_new_password"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
         android:theme="@style/EditText.Password"
-        app:layout_constraintBottom_toTopOf="@id/middle_guide"
-        app:layout_constraintStart_toEndOf="@id/start_guide"
-        app:layout_constraintEnd_toStartOf="@id/end_guide">
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        android:layout_margin="16dp"
+        >
         <EditText
             android:id="@+id/text_new_password"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:hint="@string/msg_new_password"
-            android:inputType="textPassword" />
+            android:inputType="textPassword"
+            />
     </android.support.design.widget.TextInputLayout>
 
     <android.support.design.widget.TextInputLayout
         android:id="@+id/layout_confirm_password"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
         android:theme="@style/EditText.Password"
-        app:layout_constraintTop_toBottomOf="@id/middle_guide"
-        app:layout_constraintStart_toEndOf="@id/start_guide"
-        app:layout_constraintEnd_toStartOf="@id/end_guide">
+        app:layout_constraintTop_toBottomOf="@+id/layout_new_password"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        android:layout_margin="16dp"
+        >
```

Comment:
This `>` could be on the same line of `android:layout_margin="16dp"`

Reply:
@filipedelimabrito I will keep this in mind next time.